bolt-design-system/bolt
# Add display flex to bolt list component

## Jira

http://vjira2:8080/browse/BDS-707

## Summary

Adding possibility to set display prop to flex that gives an opportunity to set dividers horizontal and vertical on items.

## Details

I'm not 100% sure that @mikemai2awesome want to do this in this way but i don't see any other way to do that.

## How to test

Run this code locally. Go to: `Components -> List -> View All` and check if all variations on flex list are working correctly.
